README.md

```diff
@@ -2,7 +2,9 @@
 To run a given script:
 ./run.sh \<directory\>/\<script_name.py\> arguments
 
-In order to run any script you will need to ensure that the Google Cloud SDK is installed: https://cloud.google.com/sdk/downloads
-
-## Authorization
-Before running a script you will need to authorize the to set the Application Default Credentials.  You can do this by running the following command: ```gcloud auth application-default login```
+## Prerequisites
+* Install Python
+* Install Pip (run `sudo easy_install pip`)
```

Comment:
right now this also relies upon Python 2.7 (if nothing else 2, not 3) - we should probably clearly state that